AcsEmulator/AcsEmulatorAPI/Models/CallAutomationModels.cs

```diff
@@ -1,4 +1,6 @@
 ﻿namespace AcsEmulatorAPI.Models
 {
-    public record CreateCallRequest(string CallbackUri);
+    public record CreateCallRequest(string CallbackUri, CommunicationIdentifierModel[] Targets);
+
+    public record PhoneNumberIdentifierModel(string value);
```

Comment:
We already have this model, please use the existing `PhoneNumber` in CommunicationIdentifier.cs